modules/tensor_mechanics/src/materials/IsotropicPlasticityStressUpdate.C

```diff
@@ -24,8 +24,8 @@ InputParameters validParams<RecomputeRadialReturnIsotropicPlasticity>()
 }
 
 
-RecomputeRadialReturnIsotropicPlasticity::RecomputeRadialReturnIsotropicPlasticity(const InputParameters & parameters) :
-    RecomputeRadialReturn(parameters),
+IsotropicPlasticityStressUpdate::IsotropicPlasticityStressUpdate(const InputParameters & parameters) :
+    RadialReturnStressUpdate(parameters),
     _yield_stress_function(isParamValid("yield_stress_function") ? &getFunction("yield_stress_function") : NULL),
     _yield_stress(getParam<Real>("yield_stress")),
```

Comment:
Turns out this change is not trivial to implement because of the inheriting classes and the use of timeDerivative on the hardening function.  I'd like to call scope creep on this change and address the function/ constant problem in a separate PR if that's acceptable.